packages/knip/src/plugins/linthtml/index.ts

```diff
@@ -0,0 +1,40 @@
+import { basename, isInternal } from '../../util/path.js';
+import { timerify } from '../../util/Performance.js';
+import { hasDependency, load } from '../../util/plugin.js';
+import type { PluginConfig } from './types.js';
+import type { IsPluginEnabledCallback, GenericPluginCallback } from '../../types/plugins.js';
+
+// https://linthtml.vercel.app/
+
+export const NAME = 'LintHTML';
+
+export const PACKAGE_JSON_PATH = 'linthtmlConfig';
+
+/** @public */
+export const ENABLERS = ['@linthtml/linthtml'];
+
+export const isEnabled: IsPluginEnabledCallback = ({ dependencies }) => hasDependency(dependencies, ENABLERS);
+
+export const CONFIG_FILE_PATTERNS = [
+  '.linthtmlrc',
+  '.linthtmlrc.json',
+  '.linthtmlrc.yml',
+  '.linthtmlrc.{js,cjs}',
+  'package.json',
+];
+
+const findPluginDependencies: GenericPluginCallback = async (configFilePath, options) => {
+  const { manifest } = options;
+
+  const localConfig: PluginConfig | undefined =
+    basename(configFilePath) === 'package.json' ? manifest.linthtmlConfig : await load(configFilePath);
+
+  if (!localConfig) return [];
+
+  const extend = localConfig.extends ? [localConfig.extends].flat().filter(extend => !isInternal(extend)) : [];
+  const plugins = localConfig.plugins ? [localConfig.plugins].flat().filter(plugin => !isInternal(plugin)) : [];
```

Comment:
If values are `isInternal()` then we might want to return them.

So if there's no reason to ignore them, you can use e.g. `toEntryPattern(extend)` so it will be returned as e.g. `"entry:./local.js"` and then it will be added as an entry file (otherwise the file might be reported as unused).

Reply:
Hey @webpro 
Okay, I will take a look